app/assets/stylesheets/donor.css.scss

```diff
@@ -183,6 +184,7 @@
 
   .donation-key-inprogress {
     background: $donation-inprogress-key;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.